apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/Aggregations.cs

```diff
@@ -32,4 +33,73 @@ public static class Aggregations
     /// <returns>A count(*) aggregation.</returns>
     public static Aggregation Count(string alias = "") =>
         new Aggregation { Count = new Count(), Alias = alias };
+
+    /// <summary>
+    /// <para>
+    /// Sum of the values of the requested property.
+    /// </para>
+    /// <list type="bullet">
+    ///   <item>
+    ///     <description>Only numeric values will be aggregated. All non-numeric values
+    ///     including `NULL` are skipped.</description>
+    ///   </item>
+    ///   <item>
+    ///     <description>If the aggregated values contain `NaN`, returns `NaN`.</description>
+    ///   </item>
+    ///   <item>
+    ///     <description>If the aggregated value set is empty, returns 0.</description>
+    ///   </item>
+    ///   <item>
+    ///     <description>Returns a 64-bit integer if the sum result is an integer value and does
+    ///     not overflow. Otherwise, the result is returned as a double. Note that
+    ///     even if all the aggregated values are integers, the result is returned
+    ///     as a double if it cannot fit within a 64-bit signed integer. When this
+    ///     occurs, the returned value will lose precision.</description>
+    ///   </item>
+    ///   <item>
+    ///     <description>When underflow occurs, floating-point aggregation is non-deterministic.
+    ///     This means that running the same query repeatedly without any changes to
+    ///     the underlying values could produce slightly different results each
+    ///     time. In those cases, values should be stored as integers over
+    ///     floating-point numbers.</description>
+    ///   </item>
+    /// </list>
+    /// </summary>
+    /// <param name="property">Property for which the sum is to be calculated.</param>
+    /// <param name="alias">A string used to retrieve the result of this aggregation.
+    /// If not provided, Datastore will pick a default name following the format `property_incremental_id++`
```

Comment:
Nit: we've still got the backticks here.

How about:

```text
If not provided, Datastore will pick a default name with a prefix of <c>property_</c>
followed by an auto-incremented ID, e.g. <c>property_1</c>.
```

Reply:
Done